algoliasearch/src/main/java/com/algolia/search/saas/BaseAPIClient.java

```diff
@@ -62,7 +62,9 @@
     private List<String> readHosts;
     private List<String> writeHosts;
 
-    /** HTTP headers that will be sent with every request. */
+    /**
+     * HTTP headers that will be sent with every request.
+     */
```

Comment:
1. Indeed, even if you can quickly glance over the first to confirm it's only formatting and then mindfully review the other ones :)
2. Indeed, I use diff to know what happened to a line (who wrote it first, then who modified it, etc). To me `git blame` seems quite useless, as it only tells you who did the *last* change to a line... Or is it more powerful than I think?